.travis.yml

```diff
@@ -0,0 +1,50 @@
+language: python
+python:
+  - 3.9
```

Comment:
What are the main advantages with 3.11 you're excited about?

Since this needs to work with ark, I think we should start with 3.7 and 3.8. If the migration to those doesn't lead to any issues, we can keep moving forward, but my suspicion is that we're going to find dependency conflicts along the way, so we should take a more gradual approach as we modernize the codebase

Reply:
Mainly for 3.11, the improved error handling and tracebacks will make fixing something that's broken much easier. I agree, we should start gradually and move to 3.7/3.8 first.